src/js/containers/account/visualizations/AccountTimeVisualizationContainer.jsx

```diff
@@ -153,8 +223,93 @@ export class AccountTimeVisualizationSectionContainer extends React.Component {
         const xSeries = [];
         const ySeries = [];
         const allY = [];
-        if (this.state.hasFilteredObligated) {
-            // In progress
+        const hasFilteredObligated = (this.props.reduxFilters.objectClass.count() > 0)
```

Comment:
See note above; I feel like we should be able to hold this in state for this component rather than redoing the `if` every time